web-animations/timing-model/animations/canceling-an-animation.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>Canceling an animation: cancel event</title>
+<link rel="help" href="https://w3c.github.io/web-animations/#canceling-an-animation-section">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../../testcommon.js"></script>
+<body>
+<script>
+
+async_test(function(t) {
+  var anim1 = createDiv(t).animate(null, 100 * MS_PER_SEC);
+  var anim2 = createDiv(t).animate(null, 100 * MS_PER_SEC);
```

Comment:
I think we can remove anim2 and just call t.done() in the oncancel. If it doesn't get called then the test will timeout and fail.